src/bridge/views/SetKeys.js

```diff
@@ -185,6 +185,16 @@ class SetKeys extends React.Component {
               </Warning>
             : <div /> }
 
+          <CheckboxButton
+            className='mt-8'
+            onClick={ this.toggleDiscontinuity }
+            state={ state.discontinuity }
+          >
+            <div style={{cursor: 'pointer'}} onClick={ this.toggleDiscontinuity }>
+              { 'I have lost important off-chain data relating to this point and need to do a hard-reset. (For example, rebooting an Arvo ship.)' }
```

Comment:
I feel like "hard reset" is better than "hard-reset", but not super important.

Reply:
You're probably right. I'll change it. Good to go otherwise? (:

Reply:
@c-johnson please confirm?